app/config/DocumentationController.scala

```diff
@@ -22,7 +22,8 @@ import javax.inject.{Inject, Singleton}
 import play.api.http.HttpErrorHandler
 import play.api.libs.json.Json
 import play.api.mvc.{Action, AnyContent, ControllerComponents}
-import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+import uk.gov.hmrc.play.bootstrap.controller.BackendController
```

Comment:
Can you please revert back to use previous import as this import is deprecated. Info can be seen [here](https://github.com/hmrc/bootstrap-play/blob/master/bootstrap-backend-play-26/src/main/scala/uk/gov/hmrc/play/bootstrap/package.scala#L37)